src/util.py

```diff
@@ -18,6 +18,10 @@ def __init__(self, name, value):
         self.value = value
 
 
+class UnableToDM(RuntimeError):
+    pass
```

Comment:
the way we have it now, i dont think this will still raise any type of error, because it raises the `UnableToDM` error, but it doesnt actually raise anything useful for us. A caller could go `UnableToDM("User does not have DMs turned on")`, etc. generally, if we want to raise an error, we should be putting an error message to make it clear, otherwise we don't need to make this class to raise an error

Reply:
I don't really understand what the issue is. Since it inherits `RuntimeError` (previously `Exception`), it has all the attributes/methods of a `RuntimeError`. I needed this class specifically because I wanted to handle a failed DM case but still properly log any other unrelated errors as an error.

Reply:
per convo on discord an NB comment would be good